### PR TITLE
Facilities page validation Error

### PR DESCRIPTION
**Pull request by visolve-openemr**
Hello Brady,
                  The Facilities page under Administration has validation issues and its fixed and moved to Visolve's Openemr_features_enhancements on branch Facilities_show_errors_bug. The following is the commit link: https://github.com/visolve-openemr/Openemr_features_enhancements/commit/b59a640413a5b9cab0c094997ff602ed4cbf450d

Thanks,
[ViSolve](http://www.visolve.com/hc/).

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @sharonco-matrix ,
Can your group look at this since the validation code is your group's baby?
thanks,
-brady

**Comment by sharonco-matrix** [comment]
Sure!

**Comment by eyalvo-matrix** [comment]
Hi all , sorry for the late respond, we did review the code and it is indeed looks good :)

File: interface/usergroup/facilities_add.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 <link rel="stylesheet" type="text/css" href="<?php echo $GLOBALS['webroot'] ?>/library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-1-9-1/index.js"></script>
+<script src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-migrate-1.2.1.min.js"></script>
```
Comment (bradymiller) [inline interface/usergroup/facilities_add.php]: hi @visolve-openemr ,
Why is this jquery-migrate-1.2.1.min.js script needed here?

**Pull request closed without merging**